openwisp_ipam/api/views.py

```diff
@@ -42,6 +47,15 @@ def get_parent_queryset(self):
         return qs
 
 
+class ImportSubnetCSVMixin(AuthorizeCSVOrgManaged):
+    def get_csv_organization(self):
+        data = self.subnet_model._get_csv_reader(
+            self, deepcopy(self.request.FILES['csvfile'])
+        )
+        org = Organization.objects.get(name=list(data)[2][0].strip())
+        return org
```

Comment:
Why don't you just add this method to ImportSubnetView instead of making this mixin?

Why this mixin? Is this to be used also by another view?

Reply:
Done.